### PR TITLE
Weekplaner update, based on ixD meeting

### PR DESCRIPTION
**Pull request by DanielRBech**
## Description

We have made some rework of weekplan page, mostly layout. Based on comments made by ixD, some new utils have been created

## Type of PR

- Feature
- Refactor

## Related Issues/Tickets

#53 

## Checklist

- [] Completed the user story described by the issue (if applicable)
- [x] Code follows the code guidelines
- [x] Tests have been added/updated (if applicable)
- [x] Documentation has been updated (if applicable)

### CODE REVIEW

**Review by Stoldt — COMMENTED** [review]
LGTM 👍

**Review by Stoldt — DISMISSED** [review]
LGTM 👍

**Review by LokeSGJ — APPROVED** [review]

**Comment by MertzAndreas** [comment]
How's the device scaling between iPhone and iPad?

File: hooks/useWeek.ts [hunk 1]
```diff
@@ -38,14 +38,24 @@ const useWeek = (initialDate = new Date()) => {
...
+    setCurrentDate(selectedDate);
+
+    // Get the current day of the week (1 = Monday, 7 = Sunday)
+    const today = new Date();
+    const currentDayOfWeek = today.getUTCDay() || 7;
+
+    const daysOffset = currentDayOfWeek - 1;
+
+    const adjustedDate = new Date(selectedDate);
+    adjustedDate.setUTCDate(selectedDate.getUTCDate() + daysOffset);
```
Comment (MertzAndreas) [inline hooks/useWeek.ts:56]: Should have comment imo 😮

**Comment by DanielRBech** [comment]
Found at couple of request before this is ready for merch

**Pull request closed without merging**